### PR TITLE
Release 0.7.3 — fix Claude run logs collapsing to a single page

### PR DESCRIPTION
**Pull request by pbean**
## Summary

Patch release **0.7.3**. Fixes the TUI **Log** tab showing only a single page for Claude runs while codex logs displayed in full.

**Root cause:** Claude Code's new fullscreen TUI (an opt-in research preview) draws on the terminal's alternate screen (`?1049h`) and repaints in place. The Log tab emulates the tmux pane capture through pyte, which has no alternate-screen buffer, so every fullscreen repaint overwrote one screen and the view collapsed to the final frame. Line-oriented CLIs like codex scroll into history normally, so they were unaffected.

**Fix:**
- The `claude` profile now forces the classic inline/scrollback renderer via `CLAUDE_CODE_DISABLE_ALTERNATE_SCREEN=1` in `[env]` — the documented override that beats any `tui` setting (user/project/local/seeded) and keeps output in native scrollback, so the capture scrolls and the whole session is reconstructable.
- Defense-in-depth: `LogView` detects an alternate-screen switch (scanning the live stream and, on cold open, the tail-seek-skipped prefix) and the dashboard flags such a capture as showing only the final frame, pointing at the agent's full JSONL transcript.

## Verified
- Diagnosis confirmed empirically against a real fullscreen run log (renders to just the final frame) vs. classic-renderer logs (full history).
- New tests: `claude` profile forces the classic renderer; four `LogView` alt-screen cases (live detection, cold-open-past-tail-seek, truncation-reset, negative).
- Full suite green (1070 passed); `trunk check` clean.

## Release prep
Version stamped to 0.7.3 across `__init__.py`/`pyproject.toml`/`module.yaml`/marketplace/`uv.lock`, CHANGELOG curated, and `docs/images` regenerated (TUI changed since v0.7.2).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved the Log tab for Claude runs by detecting alternate-screen (full-screen TUI) captures and showing a clearer “partial frame” notice instead of a misleading single-page view.
* **New Features**
  * Updated the built-in Claude profile to disable alternate-screen TUI behavior to keep log rendering consistent.
  * When available, the log pane can now surface the related transcript and provide context for the displayed output.
* **Tests**
  * Added coverage for alternate-screen detection, prefix scanning behavior, truncation/reset behavior, and tail/cold-open edge cases.

### CODE REVIEW

File: src/automator/tui/data.py [hunk 1]
```diff
@@ -457,6 +500,13 @@ def read_new(self) -> bool:
             self._offset = max(0, size - self.max_bytes)
             # offsets at or before the tail seek clamp to the first line
             self._checkpoints = [(self._offset, 0)]
+            # The render only emulates the max_bytes tail, but an altscreen switch
+            # is a whole-session property whose markers often sit in the skipped
+            # prefix (enter at session start). Scan that prefix once on open so a
+            # cold-opened fullscreen log is still flagged; the tail is covered by
+            # the per-chunk scan below.
+            if self._offset > 0 and not self.altscreen_seen:
+                self._scan_prefix_for_altscreen(self._offset)
```
[the remark this answers is not in the document]
Comment (pbean) [reply src/automator/tui/data.py]: Addressed in 6e3ce94, with a deviation worth flagging.

The core concern is valid: the cold-open prefix scan read `[0, self._offset)` — up to nearly the whole file on a large finished log (worst case ~28 MB on the biggest captures here), defeating the `max_bytes` cap.

I did **not** bound it to the session start, though. In the one real fullscreen pane log we have, the alternate-screen enter marker sits at **~3.9 MB mid-stream** (3.9 MB of classic output precedes it), so a start-only window would miss exactly what the scan exists to catch. Instead the scan is now capped at `_ALTSCREEN_PREFIX_SCAN_CAP = 8 MiB`: bounded worst-case I/O, while still covering a realistic multi-MB preamble. A marker past the cap on an oversized log is missed only on cold open — live-watching reads from offset 0 and still catches it. Added `test_log_view_altscreen_prefix_scan_is_capped` to pin the boundary.

File: tests/test_profile.py [hunk 2]
```diff
@@ -56,6 +56,12 @@ def test_builtin_profiles_load():
         assert profiles[name].usage_grace_s == 0.0
         assert profiles[name].stop_without_result_nudges is None
         assert profiles[name].subagent_stop_without_transcript is False
+    # claude forces its classic (inline/scrollback) renderer so a pane capture is
+    # not collapsed to the final frame by the fullscreen alt-screen TUI; other
+    # profiles add no such env override
+    assert profiles["claude"].env.get("CLAUDE_CODE_DISABLE_ALTERNATE_SCREEN") == "1"
+    for name in ("codex", "gemini"):
+        assert "CLAUDE_CODE_DISABLE_ALTERNATE_SCREEN" not in profiles[name].env
```
[the remark this answers is not in the document]
Comment (pbean) [reply tests/test_profile.py]: Applied in 6e3ce94 — `copilot` added to the negative assertion so all four built-in profiles enforce the Claude-only contract.

**Comment by pbean** [comment]
augment review

File: src/automator/tui/data.py [hunk 3]
```diff
@@ -442,6 +469,31 @@ def _reset_screen(self) -> None:
...
+        _ALTSCREEN_PREFIX_SCAN_CAP), not necessarily the full prefix. Sets
+        altscreen_seen; best-effort, never raises."""
+        overlap = max(len(m) for m in _ALTSCREEN_MARKERS) - 1
+        window = 1 << 20
+        try:
+            with self.path.open("rb") as f:
+                pos = 0
+                while pos < end:
+                    f.seek(pos)
+                    buf = f.read(min(window, end - pos) + overlap)
```
[the remark this answers is not in the document]
Comment (pbean) [reply src/automator/tui/data.py]: Good catch — fixed in 8200a2f. The over-read was up to `overlap` (7) bytes; the read is now `min(window + overlap, end - pos)`, so the scan stops strictly at `end` while each non-final window still reads `window + overlap` to catch a marker straddling an internal boundary (the final window's leading boundary is already covered by the previous window's overlap).

File: src/automator/tui/data.py [hunk 4]
```diff
@@ -325,6 +325,28 @@ def _render_row(row: dict) -> Text:
...
+    b"\x1b[?1047h",
+    b"\x1b[?1047l",
+    b"\x1b[?47h",
+    b"\x1b[?47l",
+)
+# Cold-open altscreen detection scans the max_bytes-skipped prefix, but caps it so a
+# huge finished log doesn't trigger a near-whole-file read that would defeat
+# max_bytes. 8 MiB comfortably covers a multi-MB classic preamble before a TUI
+# switches to the alternate screen (observed ~3.9 MB); a marker past the cap in an
+# oversized log is missed only on cold open — live-watching reads from offset 0 and
```
[the remark this answers is not in the document]
Comment (pbean) [reply src/automator/tui/data.py]: You're right, the comment overstated it — corrected in 8200a2f. `read_new()` tail-seeks to `max(0, size - max_bytes)` on first open, so attaching to an already-large log does not read from 0. Reworded: a beyond-cap marker is missed only when the log already exceeds `max_bytes` at first open; a view attached while the log was still small reads the whole stream from offset 0 and catches it regardless.

**Comment by pbean** [comment]
augment review

**Comment by pbean** [comment]
@coderabbit review
